superset/db_engine_specs/hive.py

```diff
@@ -153,6 +158,20 @@ def convert_to_hive_type(col_type: str) -> str:
             )
         schema_definition = ", ".join(column_name_and_type)
 
+        # ensure table doesn't already exist
+        if (
+            if_exists == "fail"
+            and not database.get_df(
```

Comment:
Using Pandas seems a little heavy handed to simply check if the number of records is non-zero, though I guess it's fewer lines than having to use a cursor etc. and this code is probably rarely executed.